### ISSUE TITLE
Times Series Updated Recoveries Missing

### ISSUE THREAD

**Issue by therealcyberlord** [CSSEGISandData/COVID-19#1454]
Although the number of confirmed cases and deaths are updated in the time series folder. There is no new update to the number of recoveries. In addition, the U.S. is also missing state/county information.

**Comment by bertrandom** [comment]
Yes, all the recovery numbers are showing as 0 for me in the U.S.

**Comment by andre-schneider-fraunhofer** [comment]
Please try the new files in https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_daily_reports

**Comment by bertrandom** [comment]
That is the file I'm parsing.. you can confirm this using the `xsv` tool:

```
xsv search -s 4 'US' 03-23-2020.csv | xsv select 10 | xsv stats
field,type,sum,min,max,min_length,max_length,mean,stddev
Recovered,Integer,0,0,0,1,1,0,0
```

It's 0 across the entire Recovered column for the U.S.

**Comment by hpschlenvoigt** [comment]
The option to download all daily reports to extract the recovered cases and generate a time series by oneself is not practical. I assume there is a broad interest to add the cumulative number of recovered cases in the new ("global") time series dataset and therefore kindly ask the maintainers of this repo to implement that (again). Even better would be to another file containing the new cases, since that quantity is presumably the one why we need totals, deaths and recovered.

**Comment by Bmr55** [comment]
#1250 states they will no longer be providing recovered case numbers.

**Comment by therealcyberlord** [comment]
Hopefully, they can global recoveries to the time series folder tonight. I really appreciate the work Johns Hopkins is doing.
